docker-compose.yml

```
@@ -12,7 +12,7 @@ services:
       - POSTGRES_USER=devuser
       - POSTGRES_PASSWORD=changeme
     healthcheck:
-      test: ["CMD-SHELL", "pg_isready -U devuser -d devdb"]
+      test: [ "CMD-SHELL", "pg_isready -U devuser -d devdb" ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)